requirements/fabric/base.txt

```diff
@@ -7,3 +7,4 @@ fsspec[http]>2021.06.0, <2023.5.0
 packaging >=17.1, <=23.0
 typing-extensions >=4.0.0, <=4.4.0
 lightning-utilities >=0.8.0, <0.10.0
+torchmetrics >=0.7.0, <1.1.0  # needed for using fixed compare_version
```

Comment:
This should probably go into fabric/test.txt because that's the only place we need it, right?